src/xrEngine/mailSlot.cpp

```diff
@@ -26,8 +26,8 @@ void msRead(void)
 {
 #ifndef LINUX // FIXME!!!
     DWORD cbMessage, cMessage, cbRead;
-    BOOL fResult;
-    LPSTR lpszBuffer;
+    bool fResult;
```

Comment:
Let's leave it as is.
```suggestion
    BOOL fResult;
```